modules/local/juicer_pre.nf

```diff
@@ -0,0 +1,32 @@
+process JUICER_PRE {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::yahs:1.2a.2--h7132678_0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/yahs:1.2a.2--h7132678_0' :
+        'quay.io/biocontainers/yahs:1.2a.2--h7132678_0' }"
+
+    input:
+    tuple val(meta), path(binary), path(agp), path(fai)
+
+    output:
+    tuple val(meta), path("*alignments_sorted.txt"), emit: pairs
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+
+    script:
+    def args = task.ext.args ?: ''
+    def args2 = task.ext.args2 ?: ''
+    """
+    juicer pre ${binary} ${agp} ${fai} ${args} | ${args2} > alignments_sorted.txt
```

Comment:
Same as above. Should this be a nf-core module?

Reply:
`juicer pre` is a wrap-around over [juicer tools](https://github.com/aidenlab/juicer/wiki/Download) provided by yahs. For our purpose it's better to stick to this tightly bound to yahs implementation of the wrapper which means using yahs singularity/docker/bioconda images. In that case to make this dependency explicit `juicer/pre` should be a part of the yahs module. But then I'm not sure how this fits the structure of the nf-core modules, as it can't be yahs/juicer/pre - it should still be juicer/pre for installation. Does it make sense?

Reply:
Yes lets leave it as a local module for now. We can re-evaluate after the pipeline gets a few production runs in.